Add global page variable documentation (#4)
easolhq/easolhq.github.io: docs/reference/objects/page.md

---
layout: default
title: Page

> nav_order: 2

Reviewer: I actually meant to go through and delete all of these, we shouldn't need to specify then it'll do alphabetical

parent: Objects
---

> Page is a _global_ variable, meaning it should always be accessible to you when developing on canvas

Reviewer: ```suggestion
Page is a _global_ variable, meaning it should always be accessible to you when developing themes on Easol
```
Canvas is dead

> If a user visits https://unplugged.rest/shop/koya-2021-10-09 for example

Reviewer: Can we use a dummy example here rather than a live one

The following will be available in the templates:

# page.url

The full url of the request

`=> https://unplugged.rest/shop/koya-2021-10-09`

# page.host

> Just the host of the request

Reviewer: ```suggestion
The request host
```

`=> unplugged.rest`

# page.path

> The absolute path of the request

Reviewer: ```suggestion
The relative path of the request
```
